pkgs/development/androidndk-pkgs/androidndk-pkgs.nix

```diff
@@ -54,8 +56,15 @@ rec {
       mkdir -p $out/bin
       for prog in ${ndkBinDir}/${targetInfo.triple}-*; do
         prog_suffix=$(basename $prog | sed 's/${targetInfo.triple}-//')
-        ln -s $prog $out/bin/${stdenv.targetPlatform.config}-$prog_suffix
+        cat > $out/bin/${stdenv.targetPlatform.config}-$prog_suffix <<EOF
```

Comment:
Yes, symlinking does not work for the executables in Android NDK 18b. The executables we refer to are bash scripts that load executable from relative locations and have the expectation that they are loaded from the Android NDK directory. If we use symlinks, then it will look relative to the path of the symlinks, breaking the executable because these paths are invalid.